.circleci/config.yml

```diff
@@ -155,6 +155,60 @@ jobs:
           name: Publish package
           command: npm publish ./lib/
 
+  deployment:
+    docker:
+      - image: cimg/node:16.15
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/fun
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "cd:48:76:dd:42:15:64:18:7c:1a:31:2b:d6:ab:60:57"
+      - *checkout_fun
+      - *restore_node_modules
+      # If the yarn.lock file is not up-to-date with the package.json file,
+      # using the --frozen-lockfile should fail.
+      - run:
+          name: Install dependencies and build Verna for production
+          command: |
+            yarn install --frozen-lockfile
+            yarn build
+      - save_cache:
+          paths:
+            - ./lib/node_modules
+            - ./examples/playground/node_modules
+            - ./node_modules
+          key: v2-front-dependencies-{{ checksum "yarn.lock" }}
```

Comment:
As `deployment` job is executed after `test` job, the lib has been already built and all dependencies have been installed. Within the job you restore those outcomes through the `*restore_node_modules`. So it is useless to install and build library again.

Reply:
Well seen ! I deleted everything you quoted